.github/workflows/manual-test-php.yml

```diff
@@ -96,7 +96,7 @@ jobs:
     steps:
       - name: Send custom JSON data to Slack workflow
         if: ${{ always() && env.SLACK_WEBHOOK != '' }}
-        uses: 'OXID-eSales/github-actions/slack_status@v4'
+        uses: 'OXID-eSales/github-actions/slack_status@v5'
```

Comment:
slack_status has been deprecated in v5. Either switch to the underlying slack action or use the begin_report/add_report/generate_report logic